README.md

```diff
@@ -62,4 +62,4 @@ Please refer to `CONTRIBUTING` if you wish to contribute to this project.
 
 ## License
 
-Distributed under the GPLv3 License. See `LICENSE` for more information.
\ No newline at end of file
+Distributed under the CeCILL and AGPLv3 Licenses. See `LICENSE` for more information.
```

Comment:
Small typo here I guess: either we keep the `LICENSE`file that references both actual licenses, or we edit the second sentence there. What do you think?

Reply:
Oh strange, the Github interface is showing that you removed all lines of code in LICENSE but not that you added some!
My bad, all good on my side!